Anyone have a recommendation for the best way to fix this?  Typescript is not my forte.

```
static/js/api/useGridData.test.ts:39:7 - error TS2322: Type '{ runId: string; executionDate: string; dataIntervalStart: string; dataIntervalEnd: string; runType: string; startDate: null; endDate: null; lastSchedulingDecision: null; state: "success"; }' is not assignable to type 'DagRun'.
  Types of property 'runType' are incompatible.
    Type 'string' is not assignable to type '"backfill" | "manual" | "scheduled" | "dataset_triggered"'.

39       { state: 'success', ...commonDagRunParams },
         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/airflow/www/static/js/api/useGridData.test.ts
+++ b/airflow/www/static/js/api/useGridData.test.ts
@@ -27,6 +27,7 @@ const commonDagRunParams = {
   executionDate: '2022-01-01T10:00+00:00',
   dataIntervalStart: '2022-01-01T05:00+00:00',
   dataIntervalEnd: '2022-01-01T10:00+00:00',
+  runType: 'scheduled' as DagRun['runType'],
   startDate: null,
   endDate: null,
   lastSchedulingDecision: null,

